src/components/Component.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Header from './Header'
+
+export default function Component(props) {
```

Comment:
Same, maybe change name to something a bit more scenario related? @mattdean-digicatapult

Reply:
let's call it `AccessDemo`

Reply:
Done